mbq/atomiq/utils.py

```diff
@@ -1,3 +1,34 @@
+import time
+
+from functools import wraps
+
+
 def time_difference_ms(start_datetime, end_datetime):
     diff_in_seconds = (end_datetime - start_datetime).total_seconds()
     return round(diff_in_seconds * 1000)
+
+
+def debounce(seconds=None, minutes=None, hours=None):
+    def wrapper(func):
+        last_run = 0
```

Comment:
This is a good setting for a true debounce but given that this is more like a periodic task I think it might make sense to default this to the current time. This will prevent the consumer from immediately running a cleanup task when it starts, which will increase startup time and could potentially cause issues if the app is constantly restarting for some reason.

Reply:
There should be ways both to test `debounce` and the mgmt command and still set this to whatever we want it to be. Don't let tests dictate how stuff runs in production. Can talk through it if you'd like.

Also happy to be told you think current time is wrong, but if we agree current time is right then let's do current time and make it work with tests.